### DISCUSSION TITLE
What is 1.apk that I see in output ?

### DISCUSSION THREAD

**Discussion by adrianTNT** [iBotPeaches/Apktool#3441] [Q&A]
While unpacking an apk file with something like:

```
java -jar apktool_2.9.0.jar --frame-path /temp/ decode my_file.apk -f -o /temp/my_apk_unpacked/
```

I see this line in the output:

```
I: Loading resource table from file: /temp/1.apk
```
- What is the `1.apk` file ? Does it show as an error or just a log ?
- Is that generated as partial output of each unpacked apk file ? 
- Meaning ... is it a problem **if two apk files unpack at the same time with same** `--frame-path` ? 
- Will there be a conflict with that `1.apk` file ?

Thank you.

**Comment by iBotPeaches** [comment]
* Its the internal framework file copied out of the Apktool jar into a location for reference by aapt/aapt2 during compilation. (https://apktool.org/docs/in-depth/frameworks)
* Its only generated once if the file doesn't exist, however tagging or redirection of frame-path location may change this. Future version of Apktool may understand what version of the framework exists to prompt updates
* I haven't actually tested if concurrency becomes an issue in a situation where the framework files does not yet exist. So let me know.
* I don't know
